ftw/usermigration/globalroles.py

```diff
@@ -0,0 +1,34 @@
+from operator import itemgetter
+from Products.CMFCore.utils import getToolByName
+
+
+def migrate_globalroles(context, mapping, mode='move'):
+    """Migrate global roles in portal_role_manager."""
+
+    # Statistics
+    moved = []
+
+    if mode != 'move':
+        raise Exception(
```

Comment:
You raise `NotImplementedError` here in another PR, i like that better. I think we should not raise the base `Exception` class. When used in  a `try` `except` catching `Exception` could possibly hide other exceptions.